src/linked_list.py

```diff
@@ -48,35 +48,38 @@ def size(self):
         return count
 
     def __len__(self):
-        """Function binds LinkedList to __len__ builtin so len() works."""
+        """Binds .size to __len__ builtin so len() works."""
```

Comment:
You're not technically binding anything to anything here. All you're doing is *return*ing the size method when the `len()` built-in is called.